.github/workflows/test_branches.yml

```diff
@@ -50,15 +50,15 @@ jobs:
           python: 3.9
           TARGET: win
           PYENV: conda
-          PACKAGES: glpk
+          PACKAGES: glpk scip
 
         - os: ubuntu-latest
           python: 3.9
           other: /conda
           skip_doctest: 1
           TARGET: linux
           PYENV: conda
-          PACKAGES:
+          PACKAGES: scip
```

Comment:
The standard place to put packages that should only be installed in conta environments is line 311:
```
for PKG in cyipopt pymumps; do
```

Reply:
Thx.  I made that change.